### PR TITLE
Microsoft.ReactNative SampleApps

### PR DESCRIPTION
**Pull request by jonthysell**
* Adds two new samples under pacakges - one in C# and one in C++, both built on top of the ABI surface prodvided by Microsoft.ReactNative project

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/2936)

### CODE REVIEW

**Comment by acoates-ms** [comment]
Should something be added to the CI loop to verify that these build?

**Comment by jonthysell** [comment]
> Should something be added to the CI loop to verify that these build?

Perhaps, but I'm having trouble getting packages/playground to build in the CI, and I didn't even touch that.

File: packages/microsoft-reactnative-sampleapps/windows/SampleAppCPP/SampleApp.vcxproj [hunk 1]
```diff
@@ -0,0 +1,205 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(SolutionDir)packages\Microsoft.Windows.CppWinRT.2.0.190730.2\build\native\Microsoft.Windows.CppWinRT.props" Condition="Exists('$(SolutionDir)\packages\Microsoft.Windows.CppWinRT.2.0.190730.2\build\native\Microsoft.Windows.CppWinRT.props')" />
+  <Import Project="$(SolutionDir)packages\Microsoft.Windows.CppWinRT.2.0.190712.2\build\native\Microsoft.Windows.CppWinRT.props" Condition="Exists('$(SolutionDir)\packages\Microsoft.Windows.CppWinRT.2.0.190712.2\build\native\Microsoft.Windows.CppWinRT.props')" />
```
Comment (micahl) [inline packages/microsoft-reactnative-sampleapps/windows/SampleAppCPP/SampleApp.vcxproj]: Nit: Dunno how this stuck around after updating the CppWinRT version, but this line could be removed.

File: packages/microsoft-reactnative-sampleapps/windows/SampleAppCS/SampleAppCS.csproj [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+      <SubType>Designer</SubType>
+    </ApplicationDefinition>
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform">
+      <Version>6.2.8</Version>
+    </PackageReference>
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\..\node_modules\react-native-windows\Microsoft.ReactNative\Microsoft.ReactNative.vcxproj">
```
Comment (micahl) [inline packages/microsoft-reactnative-sampleapps/windows/SampleAppCS/SampleAppCS.csproj]: Does this path resolve correctly?

Comment (jonthysell) [reply packages/microsoft-reactnative-sampleapps/windows/SampleAppCS/SampleAppCS.csproj]: yep

**Pull request merged**